app/assets/javascripts/components/form_page.es6.jsx

```diff
@@ -0,0 +1,20 @@
+class FormPage extends Component {
+
+  static get defaultState() {
+    
```

Comment:
When creating placeholder `static get`s, be sure to have `return {};`. This returns an empty object instead of the instance having an `undefined field`.